DEBUG not working with webpack resolver?

I'm having trouble integrating `eslint-import-resolver-webpack`. I'm assuming it's something with the way my `webpack.config.js` is set up because everything is complaining. I tried doing:

```
DEBUG=eslint-plugin-import:* ./node_modules/.bin/eslint src/atoms/textInput/docs/Page.js
```

But it doesn't provide any info:

```
/Users/ben.ilegbodu/eventbrite/eventbrite_design_system/src/atoms/textInput/docs/Page.js
   1:19  error  Unable to resolve path to module 'react'                                              import/no-unresolved
   3:27  error  Unable to resolve path to module 'foundation/containers/documentation/Documentation'  import/no-unresolved
   4:25  error  Unable to resolve path to module '_documentation/CodeExample'                         import/no-unresolved
   6:24  error  Unable to resolve path to module './README'                                           import/no-unresolved
   8:27  error  Unable to resolve path to module '!raw!../TextInput'                                  import/no-unresolved
  10:40  error  Unable to resolve path to module '!raw!./TextInputSimpleExample'                      import/no-unresolved
  11:36  error  Unable to resolve path to module './TextInputSimpleExample'                           import/no-unresolved
  13:39  error  Unable to resolve path to module '!raw!./TextInputErrorExample'                       import/no-unresolved
  14:35  error  Unable to resolve path to module './TextInputErrorExample'                            import/no-unresolved
  16:42  error  Unable to resolve path to module '!raw!./TextInputDisabledExample'                    import/no-unresolved
  17:38  error  Unable to resolve path to module './TextInputDisabledExample'                         import/no-unresolved
  19:38  error  Unable to resolve path to module '!raw!./TextInputTypeExample'                        import/no-unresolved
  20:34  error  Unable to resolve path to module './TextInputTypeExample'                             import/no-unresolved
  22:48  error  Unable to resolve path to module '!raw!./TextInputCurrencySymbolExample'              import/no-unresolved
  23:44  error  Unable to resolve path to module './TextInputCurrencySymbolExample'                   import/no-unresolved
  25:38  error  Unable to resolve path to module '!raw!./TextInputIconExample'                        import/no-unresolved
  26:34  error  Unable to resolve path to module './TextInputIconExample'                             import/no-unresolved
  28:46  error  Unable to resolve path to module '!raw!./TextInputEventHandlerExample'                import/no-unresolved
  29:42  error  Unable to resolve path to module './TextInputEventHandlerExample'                     import/no-unresolved
  31:41  error  Unable to resolve path to module '!raw!./TextInputMaxCharExample'                     import/no-unresolved
  32:37  error  Unable to resolve path to module './TextInputMaxCharExample'                          import/no-unresolved
  34:28  error  Unable to resolve path to module '../constants'                                       import/no-unresolved
```

Any ideas? I figure if I could get the debug information I could figure out what was going on.

Miscellaneous info:

- `eslint-import-resolver-webpack`: ^0.7.0
- `webpack`: ^1.13.0
- `eslint-plugin-import`: ^2.2.0

`.eslint.json`:

```json
{
    "extends": [
        "eventbrite-react",
        "plugin:import/errors",
        "plugin:import/warnings"
    ],
    "plugins": [
        "import"
    ],
    "settings":{
        "import/resolver": "webpack"
    }
}
```

If I use the default `node` resolver instead, I do get debug information and it only complains about the special loaders. Importing `react`, for instance is no longer an error.

My webpack.config.js is a sibling of package.json FWIW. Also webpack is working normally 😄 .

Thanks for any help!

--- a/resolvers/webpack/package.json
+++ b/resolvers/webpack/package.json
@@ -33,6 +33,7 @@
     "debug": "^2.2.0",
     "enhanced-resolve": "~0.9.0",
     "find-root": "^0.1.1",
+    "has": "^1.0.1",
     "interpret": "^1.0.0",
     "is-absolute": "^0.2.3",
     "lodash.get": "^3.7.0",

